### ISSUE TITLE
Send legislator emails via ActionNetwork

### ISSUE THREAD

**Issue by kevincianfarini** [climatechangemakers/climate-changemakers-act#36]
Relevant documentation: https://actionnetwork.org/docs/v2/

**Comment by kevincianfarini** [comment]
Might be worth avoiding action network altogether and use https://www.house.gov/doing-business-with-the-house/communicating-with-congress-cwc

**Comment by kevincianfarini** [comment]
There's an additional senate counterpart to the above house API. 

https://www.senate.gov/senators/scwc.htm

We should almost certainly be using those over Action Network's API.

**Comment by kevincianfarini** [comment]
We got access to SCWC, waiting on access to (House) CWC. We're going to ditch Action Network. It's too complicated and does too much for our use cases. Renaming this issue. 

One interesting piece of documentation from CWC is their planned maintenance twice weekly. 

> Heads Up We perform weekly maintenance from 12 a.m. to 6 a.m. every Sunday and 5 a.m. to 7 a.m. every Wednesday. SCWC may be intermittently out during those times.

Maybe this warrants some sort of queuing service? That could be an improvement for the future later down the line. We don't need to worry about it for 1.0.

**Comment by mcchang** [comment]
Would be nice to link to the xml spec for the CWC api (I couldn't easily find it via Google?)

**Comment by mcchang** [comment]
> We got access to SCWC, waiting on access to (House) CWC. We're going to ditch Action Network. It's too complicated and does too much for our use cases. Renaming this issue.
> 
> One interesting piece of documentation from CWC is their planned maintenance twice weekly.
> 
> > Heads Up We perform weekly maintenance from 12 a.m. to 6 a.m. every Sunday and 5 a.m. to 7 a.m. every Wednesday. SCWC may be intermittently out during those times.
> 
> Maybe this warrants some sort of queuing service? (cc @mcchang) That could be an improvement for the future later down the line. We don't need to worry about it for 1.0.

Makes sense! We'll probably want to retry on transient errors like these.

**Comment by kevincianfarini** [comment]
@mcchang I can chat with you about it offline. I don't believe the documentation is accessible by the public.
